### DISCUSSION TITLE
[WiX V4] Why is DotNetCompatibilityCheckRef not allowed in Bundle?

### DISCUSSION THREAD

**Discussion by NJullienSweet** [wixtoolset/issues#7930] [Questions]
As the title says, I'm wondering why you apparently only need to declare a `DotNetCompatibilityCheck` element in a `Fragment` for a bundle, but need `DotNetCompatibilityCheckRef` for a package. When I try to add a `DotNetCompatibilityCheckRef` in a `Bundle`, I get the following error line:
```
error WIX0005: The Bundle element contains an unexpected child element 'DotNetCompatibilityCheckRef'.
```
 Am I missing something?

**Comment by robmen** [comment]
DotNetCompatibility check is only supported in MSIs and MSMs. You can see that in the documentation because of the allowed parents: https://wixtoolset.org/docs/schema/netfx/dotnetcompatibilitycheck/

DotNetCoreSdkSearch is only supported in Bundles. You can see that in the documentation because of the allowed parents: https://wixtoolset.org/docs/schema/netfx/dotnetcoresdksearch/
